packages/react-sdk/src/model/matrixRtcSessions.ts

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2025 Nordeck IT + Consulting GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { StateEvent } from '@matrix-widget-toolkit/api';
+import Joi from 'joi';
+import { RTCFocus } from '../state/communication';
+import { isValidEvent } from './validation';
+
+export const RTC_WHITEBOARD_APPID = 'net.nordeck.whiteboard.application';
```

Comment:
```suggestion
export const RTC_APPLICATION_WHITEBOARD = 'net.nordeck.whiteboard';
```
In my opinion it is fine to have a shorter value that you have defined before (as the sate event we have already).